### ISSUE TITLE
[BUG] Eval on flake h-m configuration is broken

### ISSUE THREAD

**Issue by t184256** [nix-community/nixvim#1023]
| Field        | Description |
|--------------|-------------|
| Plugin       | -  | 
| Nixpkgs      | unstable | 
| Home Manager | unstable | 

## Description

Can no longer eval flake home-manager configurations using nixvim, making it outright unusable.
See also https://github.com/nix-community/nixvim/issues/938

## Minimal, Reproducible Example (MRE)

```nix
{
  description = "nixvim + h-m + flakes bug reproducer";
  inputs = {
    nixpkgs.url = "github:NixOS/nixpkgs";
    home-manager.url = "github:nix-community/home-manager";
    nixvim.url = "github:pta2002/nixvim";
    #flake-parts.url = "github:hercules-ci/flake-parts?rev=ac5d0b2d4d51a53a1cd4a4a10d22f4a12c3fe652";  # last not broken
    #flake-parts.url = "github:hercules-ci/flake-parts?rev=3a0408e3acf785354d8a9b4074b9d4dbb0a12852";  # first broken
    #nixvim.inputs.flake-parts.follows = "flake-parts";
  };
  outputs = { self, nixpkgs, home-manager, nixvim, ... }@inputs:
  {
    homeConfigurations.test = home-manager.lib.homeManagerConfiguration {
      pkgs = import nixpkgs { system = "x86_64-linux"; };
      modules = [
        nixvim.homeManagerModules.nixvim
        {
          home.username = "nixvim";
          home.homeDirectory = "/invalid/dir";
          home.stateVersion = "23.05";
          programs.nixvim.enable = true;
          programs.home-manager.enable = true;
        }
      ];
    };
  };
}
```

```
$ nix build '.#homeConfigurations.test.activationPackage'
error: Trying to retrieve system-dependent attributes for input nixpkgs, but this input is not a flake. Perhaps flake = false was added to the input declarations by mistake, or you meant to use a different input, or you meant to use plain old inputs, not inputs'.
```

<details><summary>--show-trace</summary>

       … while retrieving system-dependent attributes for input nixpkgs

       … while evaluating anonymous lambda

       at /nix/store/5yhqhsl8x9xmhsy3ql8hi05c3zbnxx2x-source/modules/perSystem.nix:100:27:

           99|             mapAttrs
          100|               (inputName: input:
             |                           ^
          101|                 builtins.addErrorContext "while retrieving system-dependent attributes for input ${escapeNixIdentifier inputName}" (

       … from call site

       … while evaluating the attribute 'value.content'

       at /nix/store/byybjfdwhkpv18xqj7945ca755gbisc3-source/lib/modules.nix:1016:14:

         1015|     { _type = "override";
         1016|       inherit priority content;
             |              ^
         1017|     };

       … while evaluating the attribute 'value._type'

       at /nix/store/byybjfdwhkpv18xqj7945ca755gbisc3-source/lib/modules.nix:935:73:

          934|       highestPrio = foldl' (prio: def: min (getPrio def) prio) 9999 defs;
          935|       strip = def: if def.value._type or "" == "override" then def // { value = def.value.content; } else def;
             |                                                                         ^
          936|     in {

       … while evaluating anonymous lambda

       at /nix/store/byybjfdwhkpv18xqj7945ca755gbisc3-source/lib/modules.nix:834:19:

          833|           # Avoid sorting if we don't have to.
          834|           if any (def: def.value._type or "" == "order") defs''.values
             |                   ^
          835|           then sortProperties defs''.values

       … from call site

       at /nix/store/byybjfdwhkpv18xqj7945ca755gbisc3-source/lib/modules.nix:834:14:

          833|           # Avoid sorting if we don't have to.
          834|           if any (def: def.value._type or "" == "order") defs''.values
             |              ^
          835|           then sortProperties defs''.values

       … while evaluating the attribute 'values'

       at /nix/store/byybjfdwhkpv18xqj7945ca755gbisc3-source/lib/modules.nix:838:9:

          837|       in {
          838|         values = defs''';
             |         ^
          839|         inherit (defs'') highestPrio;

       … while evaluating the attribute 'optionalValue.value'

       at /nix/store/byybjfdwhkpv18xqj7945ca755gbisc3-source/lib/modules.nix:856:5:

          855|
          856|     optionalValue =
             |     ^
          857|       if isDefined then { value = mergedValue; }

       … while evaluating anonymous lambda

       at /nix/store/byybjfdwhkpv18xqj7945ca755gbisc3-source/lib/types.nix:593:29:

          592|       merge = loc: defs:
          593|         zipAttrsWith (name: defs:
             |                             ^
          594|           let merged = mergeDefinitions (loc ++ [name]) elemType defs;

       … from call site

       … while evaluating the module argument `pkgs' in ":anon-15:anon-1":

       … while evaluating anonymous lambda

       at /nix/store/byybjfdwhkpv18xqj7945ca755gbisc3-source/lib/modules.nix:506:44:

          505|       context = name: ''while evaluating the module argument `${name}' in "${key}":'';
          506|       extraArgs = builtins.mapAttrs (name: _:
             |                                            ^
          507|         builtins.addErrorContext (context name)

       … from call site

       … while realising the context of a path

       at /nix/store/iarrg3maajm8rd4rbjx3y1gr9qdi5whw-source/docs/default.nix:6:5:

            5|   pkgsDoc =
            6|     import (pkgs.applyPatches {
             |     ^
            7|       name = "nixpkgs-nixvim-doc";

       … while evaluating anonymous lambda

       at /nix/store/iarrg3maajm8rd4rbjx3y1gr9qdi5whw-source/docs/default.nix:1:1:

            1| {
             | ^
            2|   rawModules,

       … from call site

       at /nix/store/iarrg3maajm8rd4rbjx3y1gr9qdi5whw-source/flake-modules/packages.nix:8:16:

            7|   }: {
            8|     packages = import ../docs {
             |                ^
            9|       inherit rawModules pkgs;

       … while evaluating the attribute 'value'

       at /nix/store/byybjfdwhkpv18xqj7945ca755gbisc3-source/lib/modules.nix:607:52:

          606|                 (n: value:
          607|                   [{ inherit (module) file; inherit value; }]
             |                                                    ^
          608|                 )

       … while evaluating 'dischargeProperties'

       at /nix/store/byybjfdwhkpv18xqj7945ca755gbisc3-source/lib/modules.nix:896:25:

          895|   */
          896|   dischargeProperties = def:
             |                         ^
          897|     if def._type or "" == "merge" then

       … from call site

       at /nix/store/byybjfdwhkpv18xqj7945ca755gbisc3-source/lib/modules.nix:825:137:

          824|         defs' = concatMap (m:
          825|           map (value: { inherit (m) file; inherit value; }) (builtins.addErrorContext "while evaluating definitions from `${m.file}':" (dischargeProperties m.value))
             |                                                                                                                                         ^
          826|         ) defs;

       … while evaluating definitions from `/nix/store/iarrg3maajm8rd4rbjx3y1gr9qdi5whw-source/flake-modules/packages.nix, via option perSystem':

       … while evaluating anonymous lambda

       at /nix/store/byybjfdwhkpv18xqj7945ca755gbisc3-source/lib/modules.nix:824:28:

          823|         # Process mkMerge and mkIf properties.
          824|         defs' = concatMap (m:
             |                            ^
          825|           map (value: { inherit (m) file; inherit value; }) (builtins.addErrorContext "while evaluating definitions from `${m.file}':" (dischargeProperties m.value))

       … from call site

       at /nix/store/byybjfdwhkpv18xqj7945ca755gbisc3-source/lib/modules.nix:824:17:

          823|         # Process mkMerge and mkIf properties.
          824|         defs' = concatMap (m:
             |                 ^
          825|           map (value: { inherit (m) file; inherit value; }) (builtins.addErrorContext "while evaluating definitions from `${m.file}':" (dischargeProperties m.value))

       … while evaluating the attribute 'values'

       at /nix/store/byybjfdwhkpv18xqj7945ca755gbisc3-source/lib/modules.nix:937:7:

          936|     in {
          937|       values = concatMap (def: if getPrio def == highestPrio then [(strip def)] else []) defs;
             |       ^
          938|       inherit highestPrio;

       … while evaluating the attribute 'values'

       at /nix/store/byybjfdwhkpv18xqj7945ca755gbisc3-source/lib/modules.nix:838:9:

          837|       in {
          838|         values = defs''';
             |         ^
          839|         inherit (defs'') highestPrio;

       … while evaluating the attribute 'mergedValue'

       at /nix/store/byybjfdwhkpv18xqj7945ca755gbisc3-source/lib/modules.nix:844:5:

          843|     # Type-check the remaining definitions, and merge them. Or throw if no definitions.
          844|     mergedValue =
             |     ^
          845|       if isDefined then

       … while evaluating the option `perSystem.x86_64-linux.packages':

       … while evaluating the attribute 'value'

       at /nix/store/byybjfdwhkpv18xqj7945ca755gbisc3-source/lib/modules.nix:809:9:

          808|     in warnDeprecation opt //
          809|       { value = builtins.addErrorContext "while evaluating the option `${showOption loc}':" value;
             |         ^
          810|         inherit (res.defsFinal') highestPrio;

       … while evaluating anonymous lambda

       at /nix/store/byybjfdwhkpv18xqj7945ca755gbisc3-source/lib/modules.nix:242:72:

          241|           # For definitions that have an associated option
          242|           declaredConfig = mapAttrsRecursiveCond (v: ! isOption v) (_: v: v.value) options;
             |                                                                        ^
          243|

       … from call site

       at /nix/store/byybjfdwhkpv18xqj7945ca755gbisc3-source/lib/attrsets.nix:742:20:

          741|               then recurse (path ++ [name]) value
          742|               else f (path ++ [name]) value;
             |                    ^
          743|         in mapAttrs g;

       … while evaluating 'g'

       at /nix/store/byybjfdwhkpv18xqj7945ca755gbisc3-source/lib/attrsets.nix:739:19:

          738|           g =
          739|             name: value:
             |                   ^
          740|             if isAttrs value && cond value

       … from call site

       … while evaluating anonymous lambda

       at /nix/store/5yhqhsl8x9xmhsy3ql8hi05c3zbnxx2x-source/modules/transposition.nix:104:22:

          103|           mapAttrs
          104|             (system: v: v.${attrName} or (
             |                      ^
          105|               abort ''

       … from call site

       … while evaluating the attribute 'value'

       at /nix/store/byybjfdwhkpv18xqj7945ca755gbisc3-source/lib/types.nix:599:58:

          598|         # Push down position info.
          599|         (map (def: mapAttrs (n: v: { inherit (def) file; value = v; }) def.value) defs);
             |                                                          ^
          600|       emptyValue = { value = {}; };

       … while evaluating 'dischargeProperties'

       at /nix/store/byybjfdwhkpv18xqj7945ca755gbisc3-source/lib/modules.nix:896:25:

          895|   */
          896|   dischargeProperties = def:
             |                         ^
          897|     if def._type or "" == "merge" then

       … from call site

       at /nix/store/byybjfdwhkpv18xqj7945ca755gbisc3-source/lib/modules.nix:825:137:

          824|         defs' = concatMap (m:
          825|           map (value: { inherit (m) file; inherit value; }) (builtins.addErrorContext "while evaluating definitions from `${m.file}':" (dischargeProperties m.value))
             |                                                                                                                                         ^
          826|         ) defs;

       … while evaluating definitions from `/nix/store/5yhqhsl8x9xmhsy3ql8hi05c3zbnxx2x-source/modules/transposition.nix':

       … while evaluating anonymous lambda

       at /nix/store/byybjfdwhkpv18xqj7945ca755gbisc3-source/lib/modules.nix:824:28:

          823|         # Process mkMerge and mkIf properties.
          824|         defs' = concatMap (m:
             |                            ^
          825|           map (value: { inherit (m) file; inherit value; }) (builtins.addErrorContext "while evaluating definitions from `${m.file}':" (dischargeProperties m.value))

       … from call site

       at /nix/store/byybjfdwhkpv18xqj7945ca755gbisc3-source/lib/modules.nix:824:17:

          823|         # Process mkMerge and mkIf properties.
          824|         defs' = concatMap (m:
             |                 ^
          825|           map (value: { inherit (m) file; inherit value; }) (builtins.addErrorContext "while evaluating definitions from `${m.file}':" (dischargeProperties m.value))

       … while evaluating the attribute 'values'

       at /nix/store/byybjfdwhkpv18xqj7945ca755gbisc3-source/lib/modules.nix:937:7:

          936|     in {
          937|       values = concatMap (def: if getPrio def == highestPrio then [(strip def)] else []) defs;
             |       ^
          938|       inherit highestPrio;

       … while evaluating the attribute 'values'

       at /nix/store/byybjfdwhkpv18xqj7945ca755gbisc3-source/lib/modules.nix:838:9:

          837|       in {
          838|         values = defs''';
             |         ^
          839|         inherit (defs'') highestPrio;

       … while evaluating the attribute 'optionalValue.value'

       at /nix/store/byybjfdwhkpv18xqj7945ca755gbisc3-source/lib/modules.nix:856:5:

          855|
          856|     optionalValue =
             |     ^
          857|       if isDefined then { value = mergedValue; }

       … while evaluating anonymous lambda

       at /nix/store/byybjfdwhkpv18xqj7945ca755gbisc3-source/lib/types.nix:593:29:

          592|       merge = loc: defs:
          593|         zipAttrsWith (name: defs:
             |                             ^
          594|           let merged = mergeDefinitions (loc ++ [name]) elemType defs;

       … from call site

       … while evaluating the attribute 'value'

       at /nix/store/pp82crsxr4c3ipdplxpcb3pqcgdh1h4v-source/lib/types.nix:543:38:

          542|               elemType
          543|               [{ inherit (def) file; value = def'; }]
             |                                      ^
          544|             ).optionalValue

       … while evaluating 'dischargeProperties'

       at /nix/store/pp82crsxr4c3ipdplxpcb3pqcgdh1h4v-source/lib/modules.nix:896:25:

          895|   */
          896|   dischargeProperties = def:
             |                         ^
          897|     if def._type or "" == "merge" then

       … from call site

       at /nix/store/pp82crsxr4c3ipdplxpcb3pqcgdh1h4v-source/lib/modules.nix:825:137:

          824|         defs' = concatMap (m:
          825|           map (value: { inherit (m) file; inherit value; }) (builtins.addErrorContext "while evaluating definitions from `${m.file}':" (dischargeProperties m.value))
             |                                                                                                                                         ^
          826|         ) defs;

       … while evaluating definitions from `<unknown-file>':

       … while evaluating anonymous lambda

       at /nix/store/pp82crsxr4c3ipdplxpcb3pqcgdh1h4v-source/lib/modules.nix:824:28:

          823|         # Process mkMerge and mkIf properties.
          824|         defs' = concatMap (m:
             |                            ^
          825|           map (value: { inherit (m) file; inherit value; }) (builtins.addErrorContext "while evaluating definitions from `${m.file}':" (dischargeProperties m.value))

       … from call site

       at /nix/store/pp82crsxr4c3ipdplxpcb3pqcgdh1h4v-source/lib/modules.nix:824:17:

          823|         # Process mkMerge and mkIf properties.
          824|         defs' = concatMap (m:
             |                 ^
          825|           map (value: { inherit (m) file; inherit value; }) (builtins.addErrorContext "while evaluating definitions from `${m.file}':" (dischargeProperties m.value))

       … while evaluating the attribute 'values'

       at /nix/store/pp82crsxr4c3ipdplxpcb3pqcgdh1h4v-source/lib/modules.nix:937:7:

          936|     in {
          937|       values = concatMap (def: if getPrio def == highestPrio then [(strip def)] else []) defs;
             |       ^
          938|       inherit highestPrio;

       … while evaluating the attribute 'values'

       at /nix/store/pp82crsxr4c3ipdplxpcb3pqcgdh1h4v-source/lib/modules.nix:838:9:

          837|       in {
          838|         values = defs''';
             |         ^
          839|         inherit (defs'') highestPrio;

       … while evaluating the attribute 'optionalValue'

       at /nix/store/pp82crsxr4c3ipdplxpcb3pqcgdh1h4v-source/lib/modules.nix:856:5:

          855|
          856|     optionalValue =
             |     ^
          857|       if isDefined then { value = mergedValue; }

       … while evaluating anonymous lambda

       at /nix/store/pp82crsxr4c3ipdplxpcb3pqcgdh1h4v-source/lib/types.nix:539:21:

          538|         map (x: x.value) (filter (x: x ? value) (concatLists (imap1 (n: def:
          539|           imap1 (m: def':
             |                     ^
          540|             (mergeDefinitions

       … from call site

       at /nix/store/pp82crsxr4c3ipdplxpcb3pqcgdh1h4v-source/lib/lists.nix:165:32:

          164|   */
          165|   imap1 = f: list: genList (n: f (n + 1) (elemAt list n)) (length list);
             |                                ^
          166|

       … while evaluating anonymous lambda

       at /nix/store/pp82crsxr4c3ipdplxpcb3pqcgdh1h4v-source/lib/lists.nix:165:29:

          164|   */
          165|   imap1 = f: list: genList (n: f (n + 1) (elemAt list n)) (length list);
             |                             ^
          166|

       … from call site

       … while evaluating anonymous lambda

       at /nix/store/pp82crsxr4c3ipdplxpcb3pqcgdh1h4v-source/lib/types.nix:538:35:

          537|       merge = loc: defs:
          538|         map (x: x.value) (filter (x: x ? value) (concatLists (imap1 (n: def:
             |                                   ^
          539|           imap1 (m: def':

       … from call site

       … while evaluating 'merge'

       at /nix/store/pp82crsxr4c3ipdplxpcb3pqcgdh1h4v-source/lib/types.nix:537:20:

          536|       check = isList;
          537|       merge = loc: defs:
             |                    ^
          538|         map (x: x.value) (filter (x: x ? value) (concatLists (imap1 (n: def:

       … from call site

       at /nix/store/pp82crsxr4c3ipdplxpcb3pqcgdh1h4v-source/lib/modules.nix:846:59:

          845|       if isDefined then
          846|         if all (def: type.check def.value) defsFinal then type.merge loc defsFinal
             |                                                           ^
          847|         else let allInvalid = filter (def: ! type.check def.value) defsFinal;

       … while evaluating the attribute 'mergedValue'

       at /nix/store/pp82crsxr4c3ipdplxpcb3pqcgdh1h4v-source/lib/modules.nix:844:5:

          843|     # Type-check the remaining definitions, and merge them. Or throw if no definitions.
          844|     mergedValue =
             |     ^
          845|       if isDefined then

       … while evaluating the option `home.packages':

       … while evaluating the attribute 'value'

       at /nix/store/pp82crsxr4c3ipdplxpcb3pqcgdh1h4v-source/lib/modules.nix:809:9:

          808|     in warnDeprecation opt //
          809|       { value = builtins.addErrorContext "while evaluating the option `${showOption loc}':" value;
             |         ^
          810|         inherit (res.defsFinal') highestPrio;

       … while evaluating anonymous lambda

       at /nix/store/pp82crsxr4c3ipdplxpcb3pqcgdh1h4v-source/lib/modules.nix:242:72:

          241|           # For definitions that have an associated option
          242|           declaredConfig = mapAttrsRecursiveCond (v: ! isOption v) (_: v: v.value) options;
             |                                                                        ^
          243|

       … from call site

       at /nix/store/pp82crsxr4c3ipdplxpcb3pqcgdh1h4v-source/lib/attrsets.nix:742:20:

          741|               then recurse (path ++ [name]) value
          742|               else f (path ++ [name]) value;
             |                    ^
          743|         in mapAttrs g;

       … while evaluating 'g'

       at /nix/store/pp82crsxr4c3ipdplxpcb3pqcgdh1h4v-source/lib/attrsets.nix:739:19:

          738|           g =
          739|             name: value:
             |                   ^
          740|             if isAttrs value && cond value

       … from call site

       … while evaluating the attribute 'passAsFile'

       at /nix/store/pp82crsxr4c3ipdplxpcb3pqcgdh1h4v-source/pkgs/build-support/buildenv/default.nix:76:5:

           75|     # XXX: The size is somewhat arbitrary
           76|     passAsFile = if builtins.stringLength pkgs >= 128*1024 then [ "pkgs" ] else [ ];
             |     ^
           77|   }

       … while evaluating the attribute 'passAsFile' of the derivation 'home-manager-path'

       at /nix/store/pp82crsxr4c3ipdplxpcb3pqcgdh1h4v-source/pkgs/stdenv/generic/make-derivation.nix:352:7:

          351|     // (optionalAttrs (attrs ? name || (attrs ? pname && attrs ? version)) {
          352|       name =
             |       ^
          353|         let

       … while evaluating the attribute 'value'

       at /nix/store/pp82crsxr4c3ipdplxpcb3pqcgdh1h4v-source/lib/modules.nix:607:52:

          606|                 (n: value:
          607|                   [{ inherit (module) file; inherit value; }]
             |                                                    ^
          608|                 )

       … while evaluating 'dischargeProperties'

       at /nix/store/pp82crsxr4c3ipdplxpcb3pqcgdh1h4v-source/lib/modules.nix:896:25:

          895|   */
          896|   dischargeProperties = def:
             |                         ^
          897|     if def._type or "" == "merge" then

       … from call site

       at /nix/store/pp82crsxr4c3ipdplxpcb3pqcgdh1h4v-source/lib/modules.nix:825:137:

          824|         defs' = concatMap (m:
          825|           map (value: { inherit (m) file; inherit value; }) (builtins.addErrorContext "while evaluating definitions from `${m.file}':" (dischargeProperties m.value))
             |                                                                                                                                         ^
          826|         ) defs;

       … while evaluating definitions from `/nix/store/mprqgw56n05in92ha24mpc829pqqvpjx-source/modules/home-environment.nix':

       … while evaluating anonymous lambda

       at /nix/store/pp82crsxr4c3ipdplxpcb3pqcgdh1h4v-source/lib/modules.nix:824:28:

          823|         # Process mkMerge and mkIf properties.
          824|         defs' = concatMap (m:
             |                            ^
          825|           map (value: { inherit (m) file; inherit value; }) (builtins.addErrorContext "while evaluating definitions from `${m.file}':" (dischargeProperties m.value))

       … from call site

       at /nix/store/pp82crsxr4c3ipdplxpcb3pqcgdh1h4v-source/lib/modules.nix:824:17:

          823|         # Process mkMerge and mkIf properties.
          824|         defs' = concatMap (m:
             |                 ^
          825|           map (value: { inherit (m) file; inherit value; }) (builtins.addErrorContext "while evaluating definitions from `${m.file}':" (dischargeProperties m.value))

       … while evaluating the attribute 'values'

       at /nix/store/pp82crsxr4c3ipdplxpcb3pqcgdh1h4v-source/lib/modules.nix:937:7:

          936|     in {
          937|       values = concatMap (def: if getPrio def == highestPrio then [(strip def)] else []) defs;
             |       ^
          938|       inherit highestPrio;

       … while evaluating the attribute 'values'

       at /nix/store/pp82crsxr4c3ipdplxpcb3pqcgdh1h4v-source/lib/modules.nix:838:9:

          837|       in {
          838|         values = defs''';
             |         ^
          839|         inherit (defs'') highestPrio;

       … while evaluating the attribute 'mergedValue'

       at /nix/store/pp82crsxr4c3ipdplxpcb3pqcgdh1h4v-source/lib/modules.nix:844:5:

          843|     # Type-check the remaining definitions, and merge them. Or throw if no definitions.
          844|     mergedValue =
             |     ^
          845|       if isDefined then

       … while evaluating the option `home.activation.installPackages.data':

       … while evaluating the attribute 'value'

       at /nix/store/pp82crsxr4c3ipdplxpcb3pqcgdh1h4v-source/lib/modules.nix:809:9:

          808|     in warnDeprecation opt //
          809|       { value = builtins.addErrorContext "while evaluating the option `${showOption loc}':" value;
             |         ^
          810|         inherit (res.defsFinal') highestPrio;

       … while evaluating anonymous lambda

       at /nix/store/pp82crsxr4c3ipdplxpcb3pqcgdh1h4v-source/lib/modules.nix:242:72:

          241|           # For definitions that have an associated option
          242|           declaredConfig = mapAttrsRecursiveCond (v: ! isOption v) (_: v: v.value) options;
             |                                                                        ^
          243|

       … from call site

       at /nix/store/pp82crsxr4c3ipdplxpcb3pqcgdh1h4v-source/lib/attrsets.nix:742:20:

          741|               then recurse (path ++ [name]) value
          742|               else f (path ++ [name]) value;
             |                    ^
          743|         in mapAttrs g;

       … while evaluating 'g'

       at /nix/store/pp82crsxr4c3ipdplxpcb3pqcgdh1h4v-source/lib/attrsets.nix:739:19:

          738|           g =
          739|             name: value:
             |                   ^
          740|             if isAttrs value && cond value

       … from call site

       … while evaluating the attribute 'data'

       at /nix/store/mprqgw56n05in92ha24mpc829pqqvpjx-source/modules/lib/dag.nix:83:9:

           82|         name = n;
           83|         data = v.data;
             |         ^
           84|         after = v.after ++ dagBefore dag n;

       … while evaluating the attribute 'data'

       at /nix/store/mprqgw56n05in92ha24mpc829pqqvpjx-source/modules/lib/dag.nix:89:37:

           88|     in if sorted ? result then {
           89|       result = map (v: { inherit (v) name data; }) sorted.result;
             |                                     ^
           90|     } else

       … while evaluating 'mkCmd'

       at /nix/store/mprqgw56n05in92ha24mpc829pqqvpjx-source/modules/home-environment.nix:654:17:

          653|       let
          654|         mkCmd = res: ''
             |                 ^
          655|             _iNote "Activating %s" "${res.name}"

       … from call site

       … while evaluating the attribute 'text' of the derivation 'activation-script'

       at /nix/store/pp82crsxr4c3ipdplxpcb3pqcgdh1h4v-source/pkgs/stdenv/generic/make-derivation.nix:352:7:

          351|     // (optionalAttrs (attrs ? name || (attrs ? pname && attrs ? version)) {
          352|       name =
             |       ^
          353|         let

       … while evaluating the attribute 'buildCommand' of the derivation 'home-manager-generation'

       at /nix/store/pp82crsxr4c3ipdplxpcb3pqcgdh1h4v-source/pkgs/stdenv/generic/make-derivation.nix:352:7:

          351|     // (optionalAttrs (attrs ? name || (attrs ? pname && attrs ? version)) {
          352|       name =
             |       ^
          353|         let 
</details>

**Comment by traxys** [comment]
Is this fixed with #1102 ?

**Comment by t184256** [comment]
sorry, I've missed the notification. yes, looks like it's fixed now.
